grafana/resource_data_source.go

```diff
@@ -430,36 +435,36 @@ func makeDataSource(d *schema.ResourceData) (*gapi.DataSource, error) {
 
 func makeJSONData(d *schema.ResourceData) gapi.JSONData {
 	return gapi.JSONData{
-		AssumeRoleArn:           d.Get("json_data.0.assume_role_arn").(string),
-		AuthType:                d.Get("json_data.0.auth_type").(string),
-		AuthenticationType:      d.Get("json_data.0.authentication_type").(string),
-		ClientEmail:             d.Get("json_data.0.client_email").(string),
-		ConnMaxLifetime:         int64(d.Get("json_data.0.conn_max_lifetime").(int)),
-		CustomMetricsNamespaces: d.Get("json_data.0.custom_metrics_namespaces").(string),
-		DefaultProject:          d.Get("json_data.0.default_project").(string),
-		DefaultRegion:           d.Get("json_data.0.default_region").(string),
-		Encrypt:                 d.Get("json_data.0.encrypt").(string),
-		EsVersion:               int64(d.Get("json_data.0.es_version").(int)),
-		GraphiteVersion:         d.Get("json_data.0.graphite_version").(string),
-		HTTPMethod:              d.Get("json_data.0.http_method").(string),
-		Interval:                d.Get("json_data.0.interval").(string),
-		LogLevelField:           d.Get("json_data.0.log_level_field").(string),
-		LogMessageField:         d.Get("json_data.0.log_message_field").(string),
```

Comment:
Looks like `LogMessageField` got accidentally removed.

Reply:
Oups... Sorry for that :innocent: